lib/puppet/functions/get.rb

```diff
@@ -102,7 +102,7 @@
 #   notice("Walked path is ${error.details['walked_path']}")
 # }
 # ```
-# Would notice `Walked path is [2, color]`
+# Would notice ``Walked path is [2, color]``
```

Comment:
did you mean to create double backticks?

Reply:
Gaaah, no. Good catch. It's a VSCode thing that I need to turn off